le_utils/constants/file_formats.py

```diff
@@ -41,7 +41,7 @@
 
 # constants for HTML5 zip format
 HTML5 = "zip"
-HTML5_MIMETYPE = "application/zip"
+HTML5_MIMETYPE = ".zip"
```

Comment:
This isn't really a mimetype, so we should examine our naming scheme at some point.